release_tox.ini

```diff
@@ -36,10 +36,10 @@ commands_pre =
     --only-binary 'euphonic'
 commands = {[testenv]test_command}
 
-[testenv:pypi-py38-min]
+[testenv:pypi-py310-min]
 install_command = python -m pip install --force-reinstall {opts} {packages}
 deps =
-    numpy==1.19.5
+    numpy==1.21.3
```

Comment:
The `docs/requirements.txt` still has 1.19.5 as its minimum version, is this fine to push up?